README.md

```diff
@@ -129,7 +129,7 @@ be observed with your installed package.
 
 The setup script will install three binaries. They are all identical, but
 provide convenience aliases for pwncat. The three binaries are: `pwncat`,
-`pc` and `pcat`.
```

Comment:
`pcat` was not being created by the `setup.py` script, possibly due to the framework overhaul